dEdX/src/dEdxAnalyser.cc

```diff
@@ -779,7 +983,31 @@ void dEdxAnalyser::end()
     }
   }
 
-  _ResElectrons->Fit("gaus","Q");
+  _NormLambdaFullHist[_nPart]->FitSlicesY(gausfitrel,0,-1,0,"QR",_FitLambdaFull[_nPart]);  // angular distribution, no nHits cut
+  TH1* FitLambdaFull_mean  = (TH1D*)_FitLambdaFull[_nPart]->At(1);
+  TH1* FitLambdaFull_sigma = (TH1D*)_FitLambdaFull[_nPart]->At(2);
+  for (int d=1; d<=18; d++)
```

Comment:
Where does the `18` come from? Can this be given a name or at least be derived from another variable?